src/Components/DisplayedChart/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Header } from 'semantic-ui-react'
+import { DisplayedChartStyled } from './DisplayedChartStyled'
+
+const Placeholder = () => {
+  //inline styled only for placeholder
+  return (
+    <div
+      style={{
+        height: '80%',
+        width: '80%',
+        backgroundColor: 'teal',
+        textAlign: 'center',
+        alignSelf: 'center'
+      }}
+    >
```

Comment:
Now that we're using `styled-components`, should styling go in `DisplayedChartStyled.js` instead?

Reply:
Oh, is it because this is a placeholder element and we're not concerned without its appearance? Derp